<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWRunFiles</class>
 <widget class="QWidget" name="MWRunFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>34</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="textLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="fileEditor">
     <property name="maxLength">
       <number>500000</number>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="entryNum">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>40</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="numEntries">
     <property name="text">
      <string>/?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="valid">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>10</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>0</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>118</red>
           <green>116</green>
           <blue>108</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="browseIco">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../Images/images.qrc">
       <normaloff>:/Icons/LoadFile.png</normaloff>:/Icons/LoadFile.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="browseBtn">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="liveButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Enabled if a connection can be made to your default instrument</string>
     </property>
     <property name="text">
      <string>Use live stream</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionDialog">
   <property name="icon">
    <iconset>
     <normaloff>LoadFile.png</normaloff>LoadFile.png</iconset>
   </property>
   <property name="text">
    <string>File Dialog</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../Images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>liveButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileEditor</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>liveButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>entryNum</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>liveButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>numEntries</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>liveButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>valid</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>liveButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseIco</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>liveButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseBtn</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
